src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  --header-link-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+img {
+  display: block;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
Don't use tag names for styling (except html and body)
<img width="833" alt="image" src="https://github.com/user-attachments/assets/5dad77dd-4f1a-472d-8c38-20bedf9f78f5">